tests/storage/datavolume.go

```diff
@@ -1065,10 +1065,10 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 			dvName := vm.Spec.DataVolumeTemplates[0].Name
 
 			if gcAnnotation != "" {
-				dv, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(vm.Namespace).Get(context.TODO(), dvName, metav1.GetOptions{})
+				ann := map[string]string{"cdi.kubevirt.io/storage.deleteAfterCompletion": gcAnnotation}
```

Comment:
I think nothing stops us from creating the DV with this annotation upfront, but could be wrong

Reply:
I sure tried adding the annotation just before the VM Create(), but it had no effect. Any guess why?
```
vm.Spec.DataVolumeTemplates[0].Annotations = map[string]string{"cdi.kubevirt.io/storage.deleteAfterCompletion": gcAnnotation}
```

Reply:
It's by design. See the description of the test using this:
```
[test_id:8571]GC is disabled, and after VM creation, GC is enabled and DV is annotated
```